contractcourt/channel_arbitrator.go

```diff
@@ -837,11 +865,29 @@ func (c *ChannelArbitrator) stateStep(
 	// to be confirmed.
 	case StateCommitmentBroadcasted:
 		switch trigger {
-		// We are waiting for a commitment to be confirmed, so any
-		// other trigger will be ignored.
+
+		// We are waiting for a commitment to be confirmed.
```

Comment:
should we remove this case now, instead move the anchor sweep out of the switch? (such that it is clear that it is executed for all triggers that don't take the state machine to a new state)